rootfs/common/INSTALL_PREFIX/scripts/logpull/collect/qca.sh

```diff
@@ -81,8 +81,8 @@ collect_qcawl()
                 collect_cmd plume $PHY peer_tx_stats $STA
                 collect_cmd plume $PHY peer_rx_stats $STA
             done
-            if iwconfig $IF | grep -q Mode:Managed; then
-                BSSID=$(iwconfig $IF | awk '/Access Point/{print $NF}')
```

Comment:
Generally, this is not backward compatible. Ie. any older system using previous SDKs may not work with this change.